liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/ServerFeatureSupport.java

```diff
@@ -191,6 +213,77 @@ private static boolean isSubModule(MavenProject potentialTopModule, MavenProject
         }
         return false;
     }
+    
+    protected List<MavenProject> getRelevantMultiModuleProjects(ProjectDependencyGraph graph) throws MojoExecutionException {
+    	getLog().debug("Resolve projects");
+    	
+        List<MavenProject> sortedReactorProjects = graph.getSortedProjects();
+        Set<MavenProject> conflicts = new LinkedHashSet<MavenProject>(); // keeps the order of items added in
+
+        // A leaf here is a module without any downstream modules depending on it
+        List<MavenProject> leaves = new ArrayList<MavenProject>();
+        for (MavenProject reactorProject : sortedReactorProjects) {
+            if (graph.getDownstreamProjects(reactorProject, true).isEmpty()) {
+            	getLog().debug("Found final downstream project: " + reactorProject.getArtifactId());
+            	
+            	if (skipConfigured(reactorProject)) {
+            		getLog().debug("Skip configured - skipping");
```

Comment:
Might be good to include which project is being skipped in the debug log. Same for the jar project in the next else/if.